recipe/meta.yaml

```diff
@@ -77,6 +77,24 @@ test:
   requires:
     - pip
 
+outputs:
+  - name: {{ name|lower }}{{ suffix }}
+  - name: {{ name|lower }}{{ suffix }}-cpp
+    script: install-cpp.sh
+    requirements:
+      run:
+        - protobuf
+        - {{ pin_compatible('re2') }}
```

Comment:
Do you even need `pin_compatible('re2')`? Isn't `re2` pinned globally?

Reply:
That was my first guess, but it turned out that a binary built against `onnxruntime-cpp` would fail when running because `re2 ` was missing. i.e.

```
/home/conda/feedstock_root/build_artifacts/sourcextractor_1658829706783/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/sourcextractor++: error while loading shared libraries: libre2.so.9: cannot open shared object file: No such file or directory
```

With this explicit pinning it works.